azavea/cac-tripplanner
# Feature/kak/add activities#917

## Overview

Assorted changes in Django models to support additional fields for destinations and events.

 - Adds `Activity` model to represent what's available to do at a destination/event
 - Adds flag for watershed alliance locations
 - Adds accessibility flag
 - Data migrations to set defaults for the above
 - Renames `zip` field to `zipcode`

### Demo

![accessible_activities](https://user-images.githubusercontent.com/960264/33502917-ea678676-d6af-11e7-9e44-a6c85c4726d6.png)

![image](https://user-images.githubusercontent.com/960264/33505293-cfa47d86-d6b8-11e7-8ccc-52808386542f.png)


## Notes
The `zip` model change doesn't require any changes on the front end, as the app accesses the zip code via the transformed `Postal` attribute (used to match the geocoder response format).


## Testing Instructions

 * Run migrations
 * Should see the new fields in the admin editor interface
 * Should see the new fields in the [JSON endpoint](http://localhost:8024/api/destinations/search?text=)
 * JSON response should still have `Postal` attribute set to zip code


## Checklist


Closes #917
Closes #919 
Closes #926
